### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by mmrj**
epicenter-1250 add examples to attribute manager
catch up with epicenter for minor updates (link the how-to example)

### CODE REVIEW

File: src/dom/attributes/attribute-manager.js [hunk 1]
```diff
@@ -9,6 +9,55 @@
  *
  * The Attribute Manager is specific to adding custom attributes and describing their implementation (handlers). (The [Dom Manager](../../) contains the general implementation.)
  *
+ *
+ * **Examples**
+ *
+ * Built-in attribute managers like `data-f-value` and `data-f-foreach` automatically bind variables in your project's model to particular HTML elements. However, the value of the variable itself isn't always what you want in the UI.
+ *
+ * One application of custom attribute managers is to display particular HTML elements or content based on the value of a model variable, without displaying the value of the variable itself. For example, when your model variable meets a certain condition, you may want to show specific text in your UI based on the value of that variable. Creating a custom attribute manager is an easy way to do this:
```
Comment (narenranjit) [inline src/dom/attributes/attribute-manager.js]: "content based on the value of a model variable, without displaying the value of the variable itself." is slightly confusing. Maybe just say "instead of displaying the value directly"?

File: src/dom/attributes/attribute-manager.js [hunk 2]
```diff
@@ -9,6 +9,55 @@
  *
  * The Attribute Manager is specific to adding custom attributes and describing their implementation (handlers). (The [Dom Manager](../../) contains the general implementation.)
  *
+ *
+ * **Examples**
+ *
+ * Built-in attribute managers like `data-f-value` and `data-f-foreach` automatically bind variables in your project's model to particular HTML elements. However, the value of the variable itself isn't always what you want in the UI.
```
Comment (narenranjit) [inline src/dom/attributes/attribute-manager.js]: Maybe reword with something like "However, your use-case may sometimes require "doing something" with the value of the variable, instead of just printing it out. " (also feel free to keep as-is)

Comment (narenranjit) [inline src/dom/attributes/attribute-manager.js]: `data-f-value` and `data-f-foreach` are attribute _handlers_ not managers. The manager managers handlers.

File: src/dom/attributes/attribute-manager.js [hunk 3]
```diff
@@ -9,6 +9,55 @@
...
+ *
+ *      Flow.dom.attributes.register('showProfitFeedback', '*', function(profit) {
+ *            var profitStr = '';
+ *
+ *            if (profit > 0.10) {
+ *                  profitStr += 'Great work so far! Your firm is doing extremely well.';
+ *            } else if (profit > 0) {
+ *                  profitStr += 'Your firm is profitable, but there is room for improvement.';
+ *            } else {
+ *                  profitStr += 'Careful, your firm is currently running at a loss. Consider strategies to increase sales or decrease spending.'
```
Comment (narenranjit) [inline src/dom/attributes/attribute-manager.js]: nitpick, but lets just do `message` instead of `profitStr` and `=` instead of `+=`

File: src/dom/attributes/attribute-manager.js [hunk 4]
```diff
@@ -9,6 +9,55 @@
...
+ *            }
+ *
+ *            this.html(profitStr);
+ *      });
+ *
+ * Then, you can use the attribute manager in your HTML just like other Flow.js attributes:
+ *
+ *      <div data-f-showProfitFeedback="profit"></div>
```
Comment (narenranjit) [inline src/dom/attributes/attribute-manager.js]: In this specific example I'd probably use a converter instead - 

`data-f-bind="profit | getProfitMessage"` 

I'm struggling to articulate why though :) A good rule of thumb may be "If you're happy with the _behavior_ an attribute handler does, but would like to customize the output, you'd use a custom converter". i.e., if you want to loop through an array, you're happy with how foreach works, but just want to control what's printed out you'd use a custom converter.

If you want to control the behavior, and none of the existing attribute handlers meet your needs you can define your own. For instead, maybe you want to show/hide an element based on certain model condition. For e.g.:

```
<div data-f-showif="budgetUsed, budgetAvailable" class="error">
 You've exceeded your budget!
</div>

Flow.dom.attributes.register('showif', '*', function(data){
  if (data.budgetUsed > data.budgetAvailable) {
  this.show();
 } else {
 this.hide()
}
});

```

Happy to discuss at our checkin tomorrow if this doesn't make sense.

File: src/dom/attributes/attribute-manager.js [hunk 5]
```diff
@@ -9,6 +9,55 @@
...
+ *                  schedStr += '<li><strong>' + sortedSched[i][0]
+ *                        + '</strong> currently scheduled for <strong>Month '
+ *                        + sortedSched[i][1] + '</strong></li>';
+ *            }
+ *            schedStr += '</ul>';
+ *
+ *            this.html(schedStr);
+ *      });
+ *
+ * Then, you can use the attribute manager in your HTML just like other Flow.js attributes:
```
Comment (narenranjit) [inline src/dom/attributes/attribute-manager.js]: ~~manager~~ handler

**Comment by mmrj** [comment]
@narenranjit updated based on your comments, thanks. only change outside of your direct feedback is i swapped the order of the examples so that the handler using two arguments wasn't the first one presented.
